csharp/build.sh

```diff
@@ -37,7 +37,7 @@ export CONFIGURATION=$STEP
 
 export STEP=$CONFIGURATION
 
-xbuild /p:Configuration=$CONFIGURATION $XBUILDOPT $PROJ
+xbuild /p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true $XBUILDOPT $PROJ
```

Comment:
Verified OK on CentOS 7, just add quotes: (sorry take a long time to prepare the env)
xbuild "/p:Configuration=$CONFIGURATION;AllowUnsafeBlocks=true" $XBUILDOPT $PROJ